simplematrixbotlib/match.py

```diff
@@ -111,26 +111,26 @@ def command(self, command=None):
             elif self.mention():
```

Comment:
I feel like this could be improved: `mention()` does largely the same checks we need to do here later, so it's a bit unnecessary

Reply:
I am thinking that we add a private method to the messagematch class, and implement the checks there. This would make __init__, mention, and command cleaner, reduce duplication, and be easier to maintain.